sw/host/ot_certs/src/x509/extension.rs

```diff
@@ -64,14 +106,20 @@ pub fn x509_get_ext_by_obj<'a>(x509: &'a X509, obj: &Asn1Object) -> Result<&'a A
 //     recovery (2),
 //     debug (3)
 // }
+
+// See DiceTcbInfo.
 #[derive(asn1::Asn1Read)]
-pub struct Fwid<'a> {
+struct Fwid<'a> {
     pub hash_alg: asn1::ObjectIdentifier,
     pub digest: &'a [u8],
 }
 
+// This is an internal structure used to parsed an extension using the ASN1
+// extension. We cannot use the `DiceTcbInfoExtension` in `template` since we
+// need to use specific annotations and types so that the `asn` library can
+// derive a parser.
```

Comment:
nit:
```suggestion
// This is an internal structure used to parse a DiceTcbInfo extension using the ASN1
// crate. We cannot use the `DiceTcbInfoExtension` in `template` since we
// need to use specific annotations and types so that the `asn` library can
// derive an ASN1 parser.
```